tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2093,6 +2108,11 @@ <h2>Create a distance feature</h2>
           this.isBinaryClassification_(modelType, multiClass);
       },
 
+      enablePartialDependencePlots_: function(selectedExampleAndInference, modelName, inferenceAddress) {
```

Comment:
like `areExamplesEditable_` method, it might read better if this method name is `isPartialDependencePlotsEnabled_` or something like this

Reply:
done